media/js/plugincheck/lib/plugincheck.js

```diff
@@ -0,0 +1,277 @@
+/*globals Utils */
+
+(function(exports) {
```

Comment:
hmmm, seeing it is already a global, much like Utils, perhaps just updating the JSHint bit above? or is there a benefit I am missing?